src/ui/app-root.js

```diff
@@ -1,50 +1,45 @@
-import thoughtsMindMapView from 'thought/view-thoughts/thoughts-mind-map-view';
-import thoughtStorage from 'storage/thought-storage';
-import controlsHelp from 'ui/controls-help';
-import loginPage from 'ui/login/login';
-import thoughtContentEditor from 'ui/thought-content-editor';
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {LoginPage} from 'ui/login/login';
 import './layout.css';
-
-let element;
-
-export default {render};
-
-function render(props) {
-  element = document.getElementById('app-root');
-  element.style.position = 'relative';
-  element.style.height = '100%';
-
-  switch (props.page) {
-    case 'login':
-      renderLoginPage();
-      break;
-    case 'notes':
-      renderNotesPage();
-      break;
-    default:
-      throw new Error('unknown page: ', page);
+import {NotesPage} from 'ui/notes-page';
+
+export const PAGES_ENUM = {
+  EMPTY: Symbol('empty'),
```

Comment:
What is an empty page? Why is it needed?

Reply:
When the app starts, data for app is still not prepared. So we using empty page while app decides to change it to login or main screen.

Reply:
It will be removed after sagas implementation